modules/social_features/social_comment_upload/js/comment-upload.js

```diff
@@ -6,14 +6,14 @@
     attach: function(context, setting) {
       $('.comment-attachments').once('socialCommentUpload').each(function () {
         var $content = $('> div', this).filter(function () {
-              return !$(this).hasClass('panel-heading');
-            }),
-            $header = $('> summary', this),
-            display = true,
-            handle = function () {
-              display = !display;
-              $content.toggle(display);
-            };
+            return !$(this).hasClass('panel-heading');
+          }),
+          $header = $('> summary', this),
+          display = true,
+          handle = function () {
```

Comment:
Unrelated to this PR. I feel that this will break because it binds to a global `handle` variable (missing `var` keyword). This means that any comment attachment field would always toggle the last attachment field (due to `$content` being stored in the closure)

Reply:
nevermind, we're using comma's 🤦‍♂️

Reply:
@Kingdutch could you create a separate issue for this? 
We have to use this for our SaaS release today so don't want to extend the scope to much.

Reply:
I was mistaken, it'll work as designed. We'll tackle the formatting of our Javascript somewhere else ([goalgorilla/sweet-pea](https://github.com/goalgorilla/sweet-pea/)) :)